pulp_rpm/app/viewsets.py

```diff
@@ -994,6 +1021,22 @@ class CompsXmlViewSet(viewsets.ViewSet):
     ViewSet for comps.xml Upload.
     """
 
+    # TODO: adjust this policy after upload access policy design done and in place
+    DEFAULT_ACCESS_POLICY = {
+        "statements": [
+            {
+                "action": ["list", "retrieve"],
+                "principal": "authenticated",
+                "effect": "allow",
+            },
+            {
+                "action": ["create"],
```

Comment:
How did you decide that it is enough to be authenticated on order to perform compXML upload? I don't think it is correct.
You should look at the serializer https://github.com/pulp/pulp_rpm/blob/main/pulp_rpm/app/serializers/comps.py#L161
It shows that created comps will be uploaded to the repo, it means, the user should have according repo permissions.
It also has a file that it is uploaded, but we don't manage that with RBAC now, so it can be omitted.